nautobot/docs/user-guide/administration/configuration/optional-settings.md

```diff
@@ -1352,3 +1352,87 @@ Environment Variable: `NAUTOBOT_TIME_ZONE`
 The time zone Nautobot will use when dealing with dates and times. It is recommended to use UTC time unless you have a specific need to use a local time zone. Please see the [list of available time zones](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones).
 
 Please see the [official Django documentation on `TIME_ZONE`](https://docs.djangoproject.com/en/stable/ref/settings/#time-zone) for more information.
+
+---
+
+### OTEL_PYTHON_DJANGO_INSTRUMENT
+
++++ 2.2.0
+
+Default: `False`
+
+Environment Variable: `OTEL_PYTHON_DJANGO_INSTRUMENT`
```

Comment:
Are these OpenTelemetry conventions, or should the environment variables all be prefixed with `NAUTOBOT_` for self-consistency?